roles/matrix-server/tasks/setup_synapse_ext_mautrix_telegram.yml

```diff
@@ -60,6 +63,6 @@
 
 - name: Ensure matrix-mautrix-telegram.service doesn't exist
   file:
-    path: "{{ role_path }}/templates/systemd/matrix-mautrix-telegram.service.j2"
+    path: "{{ role_path }}/templates/systemd/matrix-mautrix-telegram.service"
```

Comment:
I think this is still wrong.
Isn't it supposed to delete the remote file on the server, and not the local template file from the playbook?

Reply:
Oops I went to fast there 💃

Reply:
Actually I copied this line from `setup_corporal.yml`, maybe it should be changed there too :)

Reply:
Thanks for letting me know! I've fixed `setup_corporal.yml`.

Unless you can think of something else, we can probably merge this pull request when this file removal thing gets fixed.

Reply:
Fixed